### ISSUE TITLE
Empty Output on iOS with “TensorFlowLiteTaskAudio 0.4.3” and “TensorFlow Lite Model Maker Model”

### ISSUE THREAD

**Issue by ghashi** [tensorflow/tflite-support#933]
## TLDR:

on iOS, when calling classify (a.k.a `let results = try classifier.classify(audioTensor: inputAudioTensor)`) on a model created using [“TensorFlow Lite Model Maker”](https://www.tensorflow.org/lite/models/modify/model_maker/audio_classification), then `results.classifications[1].categories` is always empty (in my case it was expected to output maximum of 5 values)

I think there’s a bug on pod “TensorFlowLiteTaskAudio” because the same model works as expected on Android.

Sample repository to reproduce this bug : https://github.com/ghashi/tensor-flow-lite-task-audio-poc

### I created a model using "TensorFlow Lite Model Maker Model”
I followed this tutorial https://www.tensorflow.org/lite/models/modify/model_maker/audio_classification and created a .tflite model that recognizes [5 kinds of birds songs](https://www.tensorflow.org/lite/models/modify/model_maker/audio_classification#the_birds_dataset). Here’s the versions I used to create it:

```
Python 3.7
Numpy Version: 1.21.6
TensorFlow Version: 2.9.3
Model Maker Version: 0.3.4
```

Then, I integrated this model into an Android and an iOS application. I used the tensorflow [“audio classification demo” repository](https://github.com/tensorflow/examples/tree/master/lite/examples/audio_classification)  to do this.

### Android works ✅

The Android app works fine. In other words, when I execute

```kotlin
tensorAudio.load(recorder)
val output = classifier.classify(tensorAudio)
```

then:

- `output[0].categories` has a maximum of 521 values (as [expected](https://www.tensorflow.org/hub/tutorials/yamnet))
- `ouput[1].categories`  has a maximum of 5 values

the image below show a screenshot of the app (left) and a debugging session(right) that shows the values mentioned above.

![criar API que berlin + recebe foto + sexo, idade, IMC (1) (2)](https://github.com/tensorflow/tflite-support/assets/884725/2ec72c2b-3001-4265-b3a0-2af75e328b4e)

### iOS doesn't work ❌

The iOS app doesn’t work as expected. In other words, when I execute

```swift
try inputAudioTensor.load(audioRecord: audioRecord)
let results = try classifier.classify(audioTensor: inputAudioTensor)
```

then:

- `results.classifications[0].categories` has a maximum of 5 values ([[t](https://www.tensorflow.org/hub/tutorials/yamnet)he **expected value is 521**)
- `results.classifications[1].categories`  is always empty (the **expected value is 5**)

the image below show a screenshot of the app (left) and a debugging session(right) that shows the values mentioned above.

![criar API que berlin + recebe foto + sexo, idade, IMC (3) (2)](https://github.com/tensorflow/tflite-support/assets/884725/1a8a04c1-6f36-4b7c-b655-bf3cf705fb96)

ps1: I'm using `TensorFlowLiteTaskAudio 0.4.3`
ps2: the iOS app works fine when using the default yamnet model. You can see in the image below that `results.classifications[0].categories` has a maximum of 521 values (as [expected](https://www.tensorflow.org/hub/tutorials/yamnet))
    
![criar API que berlin + recebe foto + sexo, idade, IMC (4) (1)](https://github.com/tensorflow/tflite-support/assets/884725/003de773-a140-42b4-a952-3139356be411)

   

### Question

Is there a bug on the pod “TensorFlowLiteTaskAudio”? Or is there some special configuration that should be done on iOS when integrating a model generated by  [“TensorFlow Lite Model Maker”](https://www.tensorflow.org/lite/models/modify/model_maker/audio_classification)? Or should I change something when generating the model .tflite for iOS?

### Steps to Reproduce 

1. create a a model using [“TensorFlow Lite Model Maker”](https://www.tensorflow.org/lite/models/modify/model_maker/audio_classification)
2. integrate this model into an iOS app 
3. run the app
4. the model is expected to return some results but it always returns an empty array

ps: I created a repository to help you reproducing this behavior. It shows:

- [how the model .tflite is created](https://github.com/ghashi/tensor-flow-lite-task-audio-poc/commit/40466ccbd8e94c6042404bbd0e53d54e6d02531a)
- [how it is integrated on Tensor flow iOS “audio classification” demo code](https://github.com/ghashi/tensor-flow-lite-task-audio-poc/commit/b37839c117abf5b6927750ba3f5b1e1ee5092281)
- [how it is integrated on Tensor flow Android “audio classification” demo code](https://github.com/ghashi/tensor-flow-lite-task-audio-poc/commit/421b2add06d930565dfb07394a27e1a8b040d9c3)

This is the link: https://github.com/ghashi/tensor-flow-lite-task-audio-poc . If you want to use it:

1. clone it
2. create your own model using `model-generator/model_maker_audio_classification-toreport.ipynb` or just skip this step
3. run the ios app (folder `ios`)
4. select “birds”  <img width="333" alt="criar_API_que_berlin___recebe_foto___sexo__idade__IMC__Online_Whiteboard_for_Visual_Collaboration" src="https://github.com/tensorflow/tflite-support/assets/884725/dd4982ad-0a5d-4d23-8be3-c6307a62aeaf">
5. debug line 162 of `ios/AudioClassification/TFLite/AudioClassificationHelper.swift` <img width="1028" alt="AudioClassification_—_AudioClassificationHelper_swift" src="https://github.com/tensorflow/tflite-support/assets/884725/529dba74-9789-4872-adb1-0eb3868e00dc">
6. check the variable “results” on the debugger ![criar API que berlin + recebe foto + sexo, idade, IMC (3) (1)](https://github.com/tensorflow/tflite-support/assets/884725/7b21bc17-bb7d-4d0f-822f-b1b87b3b46b7)

**Comment by khanhlvg** [comment]
@priankakariatyml Can you take a look when you have time?
